plugin-maven/build.gradle

```diff
@@ -69,7 +69,9 @@ dependencies {
 		implementation "com.diffplug.spotless:spotless-lib-extra:${libVersion}"
 	}
 
-	implementation "org.codehaus.plexus:plexus-resources:${VER_PLEXUS_RESOURCES}"
+	implementation("org.codehaus.plexus:plexus-resources:${VER_PLEXUS_RESOURCES}") {
+		exclude group: 'classworld', module: 'classworld'
```

Comment:
Is it "classworld" or "classworld**s**"?
I think the chain of transitive dependencies goes like this:
```
org.codehaus.plexus:plexus-resources:jar:1.0.1:compile
+- org.codehaus.plexus:plexus-utils:jar:3.0.8:compile
\- org.codehaus.plexus:plexus-container-default:jar:1.0-alpha-9-stable-1:compile
   \- classworlds:classworlds:jar:1.1-alpha-2:compile
```

Reply:
Thanks for the heads-up. I need another look...

Reply:
I am afraid the problem I have found during investigation of #391 is sporadic, hence I assumed that my findings fixed the problem. So to conclude:
* Problem described in #391 cannot be reproduced.
* Problem found in during investigation of #391 is sporadic.
* I have no time to investigate any further. Will reword this PR and make it an enhancement.